README.md

```diff
@@ -67,4 +66,22 @@ following syntax:
 
 ```handlebars
 {{#cite my-citation-key}}
-```
\ No newline at end of file
+``````
+
+## Configuration
+
+A complete `preprocessor.bib` section with all possible options:
+
+```toml
+[preprocessor.bib]
+bibliography = "my_biblio.bib"
+renderer = ["html"]
```

Comment:
Is `renderer` actually used anywhere? I think renderer support is checked via command line arguments, not via the configuration.

Reply:
Actually no, I think I had to specify for the preprocessors but it's not required as far as I've seen.